jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -112,6 +112,17 @@ class Terminal extends React.Component {
         term.writeln('connected to temporary workspace.\n');
       }
     };
+    ws.onclose = evt => {
+      if (evt && evt.code > 1000) {
```

Comment:
What about the 1000 special number? Please add a comment.

Reply:
The comment is right below this line ;) I'll make it more explicit.